### PR TITLE
adding validation regex for id selector

### LINKED ISSUE #11650: amp-story breaks with invalid page IDs
We do not validate the IDs of `<amp-story-page>` elements, but `querySelector` does not support all forms of IDs.  I'd like to figure out what other AMP components that are dependent on IDs are doing here and match their behavior.

### PR DESCRIPTION
**Pull request by prateekbh**
Fixes #11650

- Changes `scopedQuerySelector` to `document.getElementById` in `amp-story-page.js`

cc: @alanorozco @newmuis @choumx

### CODE REVIEW

**Comment by prateekbh** [comment]
Haha, failing travis tells its working

**Comment by prateekbh** [comment]
updated the page to not use querySelector and hence no more changing any validation rules

File: extensions/amp-story/0.1/amp-story-page.js [hunk 1]
```diff
@@ -527,8 +527,8 @@ export class AmpStoryPage extends AMP.BaseElement {
     } else {
       let mediaElement;
       try {
-        mediaElement = scopedQuerySelector(
-            this.element, `#${autoAdvanceAfter}`);
+        mediaElement =
+            this.win.document.getElementById(`${autoAdvanceAfter}`);
```
Comment (erwinmombay) [inline extensions/amp-story/0.1/amp-story-page.js]: just use '' if no interpolation is occurring

**Comment by choumx** [comment]
Please update the PR title and description.

**Comment by prateekbh** [comment]
https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/0.1/amp-story-page.js

is no more referring any `amp-story` with id. do we need all other instances to be removed as well?

**Comment by newmuis** [comment]
Sorry, #11819 moved this to https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/0.1/page-advancement.js#L527

**Comment by newmuis** [comment]
But yes, if there are other instances of getting a page by its ID, we should treat it the same way everywhere.

**Pull request closed without merging**